front-end-ui/README.md

```diff
@@ -1,3 +1,143 @@
-# FrontEndUi
+# Using Acme Gifts
+
+After having gone through the setup process, you should now be at landing page (by default: https://localhost:5064). 
+
+
+## Landing page
+
+Here you have the option to login or sign up for an account. If you sign in using your twitter account, an Acme Gifts account will be automatically created with existing twitter information. Note that Acme Gifts communications use SSL and use JWTs.
+
+Lets sign up for an Acme Gifts account. 
+
+![alt text](collateral/login.bmp)
+
+
+## Sign in with twitter
+
+Before we sign up for an account, let see sign in with twitter. For that click on the **Sign in with twitter** button. This will bring up the twitter sign in page.
+
+![alt text](collateral/twitterSignInPage.bmp)
+
+Enter your twitter information and click **sign in**.  
+After a moment, you will be routed to the main Acme Gifts groups page as seen here:
+
+![alt text](collateral/groupsSignedInUsingTweeter.bmp)
+
+As we mentioned previously, Acme Gifts has already created a profile for you and you are ready to start creating groups and occasions.  
+Note that if you sign in with twitter, you must do so thereafter.
+If you rather sign in directly with Acme gifts using a username and password, lets do that next. 
```

Comment:
Possible alternative:

> You also have the option of using a username and password for authentication, but whichever way you choose when you sign up (directly or via Twitter) you will only be able to sign in that way in the future. If you would prefer to sign into Acme Gifts directly using a username and password instead of using Twitter authentication, you will need to create an account with Acme Gifts directly, instead of creating an Acme Gifts account with Twitter. We'll show that next.